### DISCUSSION TITLE
Ability to hide LG software volume control?

### DISCUSSION THREAD

**Discussion by foss-** [MonitorControl/MonitorControl#567] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

While it is possible to hide the monitor software showing and only use macOS UI to show screen brightness adjustments, I think it is not possible to do the same for volume changes.

### Describe the solution you'd like

It would be nice if it was also possible to hide monitor software showing and only use macOS UI to show volume adjustments.

### Describe alternatives you've considered

This is what is corrently shown for an LG Ultra HD monitor connected via USB-C. The monitor software is shown overlaying the macOS UI.
![LG Ultra HD monitor](https://user-images.githubusercontent.com/5708172/132210593-db41bc02-f949-4341-934d-f0e3b5a953f5.JPG)

### Anything else?

In Settings > Displays there is the option to `Disable macOS volume OSD`. This request is about an option to `Disable monitor volume OSD`.

**Comment by waydabber** [comment, marked as answer]
Hey @foss-. Some LG displays allow suppression of the volume OSD by issuing a disable OSD command shortly after changing the volume. But for most of the time this makes the volume operations slow, causes the monitor OSD to flicker (as it is still displayed briefly) and for some displays the very same command permanently disables OSD which is obviously not good :) Version 2.1.0 had this feature but it was removed from 3.0.0 due to the complaints. We added the option to hide the macOS volume OSD instead so at least there would not be two overlapping OSDs.

**Comment by foss-** [comment]
Thanks for the quick response and elaborating on the design decision. Clearly makes sense, so Disable macOS Volume OSD it is. Now waiting for https://github.com/MonitorControl/MonitorControl/issues/278 and I am a very happy camper :) Thanks so much for your work on MonitorControl which is incredibly useful.

**Reply by waydabber** [reply]
Hi, this will not make into 3.0.0 final but will be in 3.1.0. If you are impatient, you can build the app though or later there will be betas probably:

https://github.com/MonitorControl/MonitorControl/issues/278#issuecomment-914374429

**Reply by kevinjohncutler** [reply]
What's the state of hiding OSDs as of version 4.0.0? My LG volume OSDs are a hideous, bright white. I'd be willing to live with some bugs just to keep those OSDs away.

**Reply by waydabber** [reply]
Hiding LG volume OSD was removed as a function as it did not work reliably. Somewhere I had a discussion about it with somebody utilizing an alternate method but as far as I see nobody opened an issue for it and I basically forgot about that. :)

**Comment by carlosoonline** [comment]
I have the same request, thank you very much for making this app a reality.

**Reply by waydabber** [reply]
You can try some older build that had this functionality but you'll see that it does not work properly. We do not plan to reintroduce it, this is a firmware limitation of these LG display. Sorry. :(

**Comment by jamesdlow** [comment]
Selecting the "Disable macOS volume OSD" at least stops the Mac control overlapping:

<img width="842" alt="Screen Shot 2022-07-09 at 4 13 54 PM" src="https://user-images.githubusercontent.com/114436/178091182-d781e3a5-b960-4ec6-aa90-5f4cc4aaee4a.png">
